pkg/snsqs/SnsQsContext.php

```diff
@@ -53,23 +53,15 @@ public function __construct($snsContext, $sqsContext)
         } elseif (is_callable($snsContext)) {
             $this->snsContextFactory = $snsContext;
         } else {
-            throw new \InvalidArgumentException(sprintf(
-                'The $snsContext argument must be either %s or callable that returns %s once called.',
-                SnsContext::class,
-                SnsContext::class
-            ));
+            throw new \InvalidArgumentException(sprintf('The $snsContext argument must be either %s or callable that returns %s once called.', SnsContext::class, SnsContext::class));
```

Comment:
Is there any particular reason for these formatting changes? IMO they reduce readability.